1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -11,21 +11,16 @@ import { palindrome } from "./index";
 
 describe("palindrome checker", () => {
   it.each([
-    "mom",
-    "Mom",
-    "MoM",
-    "xMomx",
-    "Was It A Rat I Saw",
-    "Never Odd or Even",
-  ])("'%s' returns true", (input) => {
-    expect(palindrome(input)).toBe(true);
-  });
-
-  it("'Momx' returns false", () => {
-    expect(palindrome("Momx")).toBe(false);
-  });
-
-  it("'Never Odd or Even1' returns false", () => {
-    expect(palindrome("Never Odd or Even1")).toBe(false);
+    ["mom", true],
+    ["Mom", true],
+    ["MoM", true],
+    ["xMomx", true],
+    ["Was It A Rat I Saw", true],
+    ["Never Odd or Even", true],
+    ["Momx", false],
+    ["Never Odd or Even1", false],
+    ["1Never Odd or Even1", true],
+  ])("'%s' returns %p", (input, expected) => {
```

Comment:
Khalil recommends that we avoid using technical words in our test names, such as: true, false, returns, etc... instead, we prefer to use behavior-descriptive, non-tech words in our test names, such as
`it knows that %s is a %p, (input, expected) => {`
for that you could write "palindrome" or "non-palindrome" as the second element in the array of inputs, so it reads nicely in the console.